src/poprox_recommender/default.py

```diff
@@ -146,7 +146,8 @@ def mmr_diversification(rewards, similarity_matrix, theta, topk):
             if mr_i >= best_MR and i not in S:
                 best_MR = mr_i
                 candidate = i
-        S.append(candidate)
+        if candidate:
+            S.append(candidate)
```

Comment:
This is a tiny fix for an issue that AFAIK only comes up when you try to recommend from a a very small pool of items (like 1 item 😅 .) When there aren't any more items to add to the list of recommendations, the `candidate` was still getting appended even if it was `None`